src/main/java/org/springframework/ai/mcp/springapi/SpringIoApi.java

```diff
@@ -50,19 +59,35 @@ public Release[] getReleases(String project) {
 	}
 
 	public Generation[] getGenerations(String project) {
-		GenerationsRoot release = restClient.get()
-			.uri("https://api.spring.io/projects/" + project + "/generations")
+		GenerationsRoot release = apiClient.get()
+			.uri(uriBuilder -> uriBuilder.path("/projects/" + project + "/generations").build())
 			.accept(MediaTypes.HAL_JSON)
 			.retrieve()
 			.body(GenerationsRoot.class);
 		
 		return release._embedded.generations;
 	}
 
+	public List<UpcomingRelease> getUpcomingReleases() {
+		LocalDate start = LocalDate.now();
+		LocalDate end = start.plusDays(90L);
```

Comment:
Should be maybe make the 90 days configurable and increase the default to 180 maybe?

Reply:
Done. Added `calendar.window.days` to application.properties. Set default to 180 days as you desired.